tests/conftest.py

```diff
@@ -260,20 +261,36 @@ def pytest_addoption(parser):
     parser.addoption('--test_downloads', action='store_true', default=False)
 
 
-def marker_names(item):
+def pytest_configure(config: pytest.Config):
+    """Add filterwarnings for vtk < 9.1 and numpy bool deprecation"""
+    if pyvista.vtk_version_info < (9, 1):
+        warnings = config.getini('filterwarnings')
+        warnings.append(
+            r'ignore:.*np\.bool.{1} is a deprecated alias for the builtin .{1}bool.*:DeprecationWarning'
+        )
+
+
+def marker_names(item: pytest.Item):
```

Comment:
Added in pytest_configure for two reasons:

- To ignore warning depending on vtk version
- Because the warning is emitted during test collection, ie. before `pytest_collection_modifyitems`